data/sdsswork.cfg

```diff
@@ -271,69 +271,70 @@ sdR_N = $BOSS_SPECTRO_DATA_N/{mjd}/sdR-{br}{id}-{frame}.fit
+spField = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spField-@pad_fieldid|-{mjd}.fits
+spfibermap = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spfibermap-@pad_fieldid|-{mjd}.fits
+spZall = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/{run1d}/spZall-@pad_fieldid|-{mjd}.fits
+spZbest = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/{run1d}/spZbest-@pad_fieldid|-{mjd}.fits
+spZline = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/{run1d}/spZline-@pad_fieldid|-{mjd}.fits
+spXCSAO = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/{run1d}/spXCSAO-@pad_fieldid|-{mjd}.fits
+spSpec = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/coadd/{mjd}/spSpec-@pad_fieldid|-{mjd}-{catalogid}.fits
+specLite = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@sptypefolder|/lite/@fieldgrp|/@pad_fieldid|@isplate|/{mjd}/spec-@pad_fieldid|-{mjd}-{catalogid}.fits
+specFull = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@sptypefolder|/full/@fieldgrp|/@pad_fieldid|@isplate|/{mjd}/spec-@pad_fieldid|-{mjd}-{catalogid}.fits
+spAllField = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@sptypefolder|/full/@fieldgrp|/@pad_fieldid|@isplate|/{mjd}/spAll-@pad_fieldid|-{mjd}.fits
+spAllLineField = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@sptypefolder|/full/@fieldgrp|/@pad_fieldid|@isplate|/{mjd}/spAllLine-@pad_fieldid|-{mjd}.fits
 #boss DRP Custom Coadd Paths
-spCOADDS = $BOSS_SPECTRO_REDUX/{run2d}/{phase}_{survey}_COADDS.par
-spPlanCustom = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/spPlanCustom-{coadd}-{mjd}.par
-spAll_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spAll-{run2d}-{coadd}.fits
-spAll-lite_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spAll-lite-{run2d}-{coadd}.fits
-spAllLine_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spAllLine-{run2d}-{coadd}.fits
-spFullsky = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/spFullsky-{coadd}-{mjd}.fits
-spZall_coadd = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/{run1d}/spZall-{coadd}-{mjd}.fits
-spZbest_coadd = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/{run1d}/spZbest-{coadd}-{mjd}.fits
-spZline_coadd = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/{run1d}/spZline-{coadd}-{mjd}.fits
-spXCSAO_coadd = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/{run1d}/spXCSAO-{coadd}-{mjd}.fits
-spSpec_coadd = $BOSS_SPECTRO_REDUX/{run2d}/{coadd}/coadd/{mjd}/spSpec-{coadd}-{mjd}-{catalogid}.fits
-specLite_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/lite/{coadd}/{mjd}/spec-{coadd}-{mjd}-{catalogid}.fits
-specFull_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/full/{coadd}/{mjd}/spec-{coadd}-{mjd}-{catalogid}.fits
-spAllField_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/full/{coadd}/{mjd}/spAll-{coadd}-{mjd}.fits
-spAllLineField_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/full/{coadd}/{mjd}/spAllLine-{coadd}-{mjd}.fits
+spCOADDS = $BOSS_SPECTRO_REDUX/{run2d}/@spcoaddfolder|/{phase}_{survey}_COADDS.par
```

Comment:
Does this name need to be in all caps?  Not necessarily suggesting we change it though.

Reply:
It does not, however, the name matches the name of the data model.